pkg/apis/core/v1/defaults_test.go

```diff
@@ -78,29 +78,52 @@ func TestWorkloadDefaults(t *testing.T) {
 		".Spec.Containers[0].StartupProbe.TimeoutSeconds":                "1",
 		".Spec.Containers[0].TerminationMessagePath":                     `"/dev/termination-log"`,
 		".Spec.Containers[0].TerminationMessagePolicy":                   `"File"`,
+		".Spec.Containers[0].LivenessProbe.Handler.GRPC.Service":         `""`,
```

Comment:
I think you don't want these - they use the old name "Handler" rather than ProbeHandler  etc